lxd/resource_lxd_container_test.go

```diff
@@ -13,7 +13,7 @@ import (
 	"github.com/lxc/lxd/shared"
 )
 
-func TestAccContainer_basic(t *testing.T) {
+func TestAccContainerBasic(t *testing.T) {
```

Comment:
Just a quick style nit: the names of tests for the core Terraform providers follow the underscore format ([example](https://github.com/hashicorp/terraform/blob/master/builtin/providers/aws/resource_aws_instance_test.go)).

I don't want that to overshadow the cool stuff in this PR, though. :)